spec/features/facets_spec.rb

```diff
@@ -67,6 +67,26 @@
     expect(page).to have_css('#facet-format', visible: true) # assert that it didn't re-collapse
   end
 
+  describe 'Facet heading button focus with Firefox' do
+    before do
+      Capybara.current_driver = :selenium_headless
+      page.current_window.resize_to(1200, 700)
+    end
+
+    it 'changes to the button on button click in Firefox' do
```

Comment:
Isn't this run on chrome headless?  This decription seems misleading.

Reply:
My understanding is that `:selenium_headless` is Firefox based on the [capybara docs](https://github.com/teamcapybara/capybara#selenium). I do know that when I run this locally it'll fail if I take the new JS out. But with the new JS it passes. It will pass just fine if `:selenium_chrome_headless` is used because chrome handles button focus well. There's probably a better way of setting a custom driver per one test block, but I wasn't sure.

Reply:
Oh neat. I didn't realize it was able to do that.  I wonder if we should just always depend on Firefox rather than depend on 2 web browsers. (doesn't need to be taken up here).